StripePaymentSheet/StripePaymentSheet/Source/Analytics/PaymentSheetAnalyticsHelper.swift

```diff
@@ -255,9 +255,20 @@ final class PaymentSheetAnalyticsHelper {
             guard let elementsSession else { return nil }
             return PaymentSheet.isLinkEnabled(elementsSession: elementsSession, configuration: configuration)
         }()
+        
         var additionalParams = [:] as [String: Any]
         additionalParams["duration"] = duration
         additionalParams["link_enabled"] = linkEnabled
+        if event == .paymentSheetLoadSucceeded {
```

Comment:
Instead of checking the event type here, can we instead find where we send `paymentSheetLoadSucceeded` and add `link_mode` as one of the `params` we pass in?